test/spanner.ts

```diff
@@ -4715,13 +4715,7 @@ describe('Spanner with mock server', () => {
               2,
               'The mock commit method should have been called exactly twice.',
             );
-            const firstRequest = capturedCommitRequests[0];
```

Comment:
Why is this removed ?

Reply:
since, now we are returning precommit token in the response of executeStreamingSql. the precommit token is not null here